versions.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.4.0, < 2.0.0"
 
   required_providers {
-    aws = ">= 3.0"
+    aws = ">= 4.3.5, < 5.0.0"
```

Comment:
I'm not thrilled about this since we're already in the middle of migrating to version 5. We'll be back here soon to upgrade the maximum version. Can we just... not set maximum versions?